infrastructure/main.tf

```diff
@@ -11,6 +11,8 @@ provider "vault" {
 locals {
   app_full_name = "${var.product}-${var.component}"
   env_ase_url = "${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
+  default_default_print_url = "https://ccd-case-print-service-${env_ase_url}"
```

Comment:
I think this needs to be `http` instead of `https`. Just discovered that `SSL` is failing when using `.internal` names because it's a self-signed.